build/yamls/antrea-aks.yml

```diff
@@ -3911,9 +3911,20 @@ data:
     #tlsMinVersion:
 
     # The name of the interface on Node which is used for tunneling or routing the traffic across Nodes.
-    # If there are multiple IP addresses configured on the interface, the first one is used.
-    # The interface configured with Node IP is used if this parameter is not set.
+    # If there are multiple IP addresses configured on the interface, the first one is used. The order for
+    # configuring tunneling or routing the traffic across Nodes is (from highest to lowest):
+    # 1. transportInterface
+    # 2. transportAddressCIDR
+    # 3. The Node IP
     #transportInterface:
+
+    # The network CIDRs of the interface on Node which is used for tunneling or routing the traffic across
+    # Nodes. If there are multiple interfaces configured the same network CIDR, the first one is used. The
+    # order for configuring tunneling or routing the traffic across Nodes is (from highest to lowest):
+    # 1. transportInterface
+    # 2. transportAddressCIDR
+    # 3. The Node IP
+    #transportAddressCIDR: <IPv4 CIDR>,<IPv6 CIDR>
```

Comment:
We should name it "transportAddressCIDRs" if there can be both IPv4 and IPv6 CIDRs?

How about transportInterfaceCIDRs if it is only use case is to look up transport interfaces.

I like to hear how other people would suggest for this parameter.

Reply:
> We should name it "transportAddressCIDRs" if there can be both IPv4 and IPv6 CIDRs?
> 
> How about transportInterfaceCIDRs if it is only use case is to look up transport interfaces.
> 
> I like to hear how other people would suggest for this parameter.

Thanks for your reply, I also agress `transportAddressCIDRs` will be better, but I have noticed the k8s naming convention, such as node-ip for kublet and cluter-cidr for controller-manager,  there also set as <IPv4>,<IPv6> or <IPv4 CIDR>,<IPv6 CIDR>. Maybe the parameter is configurable string, which does not care the content.

Reply:
I guess that's because "node-ip" was single IP initially and became two IPs when K8s started to support dual-stack: https://github.com/kubernetes/kubernetes/commit/75242fce7aa8a8f9e703b8602587900ca5aaf937. cluster-cidr should have the same history too.
`transportInterfaceCIDRs` sounds good to me too.

Reply:
@tnqn Thanks for catching this, I will update it.